packages/driver/cypress/integration/commands/xhr_spec.js

```diff
@@ -2122,6 +2122,18 @@ describe('src/cy/commands/xhr', () => {
         .wrap({ foo: 'bar' }).as('foo')
         .route(/foo/, '@bar')
       })
+
+      it('throws when fixture cannot be found', (done) => {
```

Comment:
Add link to original issue this test is addressing.

```suggestion

      // https://github.com/cypress-io/cypress/issues/7818
      it('throws when fixture cannot be found', (done) => {
```